examples/pcpChannelClient/README.md

```diff
@@ -0,0 +1,93 @@
+This directory contains a tiny sample client for the pcpChannel WebSockets connection.
```

Comment:
Can the payloads which the PSP should send in case of a setting or active preference set change be also documented somewhere?